app/src/main/java/com/example/dailytaskmanager/MainActivity.java

```diff
@@ -0,0 +1,72 @@
+package com.example.dailytaskmanager;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import java.util.Calendar;
+
+import android.view.View;
+import android.widget.Button;
+import android.widget.DatePicker;
+import android.widget.EditText;
+import android.widget.Button;
+import android.widget.ImageButton;
+import android.widget.TimePicker;
+import android.app.DatePickerDialog;
+import android.app.TimePickerDialog;
+
+public class MainActivity extends AppCompatActivity implements View.OnClickListener{

```

Comment:
Rename MainActivity to something more meaningful like CreateTimeLogActivity

Reply:
Please let me know if I shall rename it to ActivityTimeLogger or as suggestion of yours.